tests/Fixtures/TestBundle/Filter/ArrayRequiredFilter.php

```diff
@@ -19,6 +19,11 @@
 
 final class ArrayRequiredFilter extends AbstractFilter
 {
+    public static function getPropertyHelperServiceName(): string
+    {
+        return '';
```

Comment:
it's not intended right?

Reply:
It is :sweat_smile: Since this filter is extending the `Orm` filter but is not really an ORM one.